kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/thirdChallenge/FlightsPair.java

```diff
@@ -0,0 +1,41 @@
+package com.kodilla.good.patterns.challenges.thirdChallenge;
+
+import java.util.Iterator;
+import java.util.List;
+
+public class FlightsPair {
```

Comment:
Standardowo skomplikowales :) 
chodzilo o to, zeby FlghtPair mialo w sobie pare lotow: z pe start do inter oraz z inter do end, metoda zwracac ma List<FlightPair>